stories/05-architecture/06-Component-Endpoint.stories.mdx

```diff
@@ -0,0 +1,56 @@
+<Meta title="Architecture|Understanding the Components Endpoint" />
+
+# Components Endpoint
+When a user hits an endpoint in Irving, the application fires a request to the CMS' API. The response is parsed by Irving and displayed to the user as rendered React components.
+
+The components API is how your CMS describes what Irving should render for any given route. Irving will make a request to your API with a ?path query param. The value of this query param is the path currently being loaded. Navigating to /about-us/ will create a request of /components?path=/about-us/. The CMS should parse this request and return a JSON response.
+
+You can also use context to describe the context of the API load. In Core Irving context=site will be passed for the initial load to indicate that defaults should also be included in the payload.
+
+## Response
+All responses follow this format,
+```json
+{
+  "defaults": [],
+  "page": [],
+  "providers": [],
+  "redirectTo": "",
+  "redirectStatus": 0
+}
+```
+
+* `defaults`: An array of components that informs Irving what components are "global" and should persist across all requests. This is good for setting up headers, footers, scripts, and other components that should exist on every page. Defaults can be passed at any point, but Irving will automatically add `context=site` to the initial request, which can be used as a flag to return the default components.
```

Comment:
Could probably add a bit more flavor here about what this means exactly. Maybe something like "This means components appearing in the `defaults` array will only render once on the initial, server-side load of your Irving site"

Reply:
Copy paste job, we'll edit later.